sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/metric/config/MetricConfig.java

```diff
@@ -42,6 +51,12 @@ public class MetricConfig implements BaseConfig {
     @ConfigFieldKey("maxTimeSeries")
     private Integer maxTimeSeries = MAXIMUM_TIME_SERIES_VALUE;
 
+    /**
+     * The metric common tag keys, multiple keys separated by commas
+     */
+    @ConfigFieldKey("common.tag.keys")
+    private String commonTagKeys = String.join(COMMA, MetricCommonTagEnum.getAllKeys());
```

Comment:
The metric.common.tag.keys configuration item is not used here.

Reply:
done